### ISSUE TITLE
Testing the new release

### ISSUE THREAD

**Issue by snmishra** [snmishra/ftd2xx#16]
@holdenweb and @kashperanto could you check that the new [RC on pypi](https://pypi.org/project/ftd2xx/1.1.0rc1/) is working for you?

**Comment by holdenweb** [comment]
Confirm it's working for me on Windows, but the next message will come from my Mac, where the picture is sadly somewhat less rosy.

**Comment by holdenweb** [comment]
Installing on MacOS fails due to a bogus dependency:
```
(general) fathead:New sholden$ pip install ftd2xx==1.1.0rc1
Looking in indexes: https://pypi.org/simple, https://pypi.python.org/simple
Collecting ftd2xx==1.1.0rc1
  Downloading https://files.pythonhosted.org/packages/2c/0a/1a4e79a0c35a1382b2128afac81fd1548d41e23656da84d3be25805398ac/ftd2xx-1.1.0rc1-py3-none-any.whl (155kB)
    100% |████████████████████████████████| 163kB 645kB/s
Requirement already satisfied: future in /usr/local/anaconda3/envs/general/lib/python3.6/site-packages (from ftd2xx==1.1.0rc1) (0.16.0)
Collecting pywin32 (from ftd2xx==1.1.0rc1)
  Could not find a version that satisfies the requirement pywin32 (from ftd2xx==1.1.0rc1) (from versions: )
No matching distribution found for pywin32 (from ftd2xx==1.1.0rc1)
You are using pip version 19.0.2, however version 19.1.1 is available.
You should consider upgrading via the 'pip install --upgrade pip' command.
```

I'm not sure why this should be, given that line 55 of setup.py specifically includes pywin32 unless `sys.platform` is equal to `"win32"` - assuming the release candidate is simply current master?

**Comment by holdenweb** [comment]
Cloning the library and running

    pip install -e .

seems to install fine, though. Sadly I then had trouble finding appropriate ftd2xx support libraries! This may have to wait awhile due to other priorities.

**Comment by snmishra** [comment]
@holdenweb Thanks for testing it. I had the wrong syntax for requirements. I have released rc2 that seems to install fine on Linux. But testing on a VM makes the VM crash. I'll leave users on Linux and Mac to their own devices (no pun intended). I'll give you another day if you want to test the rc2. Plan to release 1.1.0 this weekend.

**Comment by holdenweb** [comment]
OK, the install issue was due to the absence of libftd2xx on my Mac (reloaded the OS a few weeks ago). After following the recommended procedure in the download at https://www.ftdichip.com/Drivers/D2XX.htm (download file itself is https://www.ftdichip.com/Drivers/D2XX/MacOSX/D2XX1.4.4.dmg) I can use the library without issues. Sadly I discovered I have no ftd2xx devices available, or they aren't being detected. Consider this at least a confirmation that the library can be used.
